requirements.txt

```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
+﻿pynwb==1.3.0
+tqdm==4.45.0
+natsort==7.0.1
+numpy==1.18.2
+pandas==1.0.3
+scipy==1.4.1
+h5py==2.10.0
+PyYAML==5.3.1
+# spikeextractors==0.9.3
+spikeextractors==0.9.5
+spikesorters==0.3.0
+spiketoolkit==0.6.1
+psutil==5.7.0
+git+https://github.com/catalystneuro/roiextractors
+neo
+=======
```

Comment:
can you fix this?

Reply:
Once PR #228 introduces the GIN framework, I'll add the Neuralynx tests and update this branch in the other respects to ensure it all works well